gtsfm/utils/features.py

```diff
@@ -88,11 +88,15 @@ def convert_to_epipolar_lines(normalized_coordinates_i1: np.ndarray, i2Ei1: Esse
     Returns:
         Corr. epipolar lines in i2, of shape Nx3.
     """
-    if normalized_coordinates_i1 is None or normalized_coordinates_i1.size == 0:
+    if coordinates_i1 is None or coordinates_i1.size == 0:
         return None
 
-    epipolar_lines = convert_to_homogenous_coordinates(normalized_coordinates_i1) @ i2Ei1.matrix().T
-    return epipolar_lines
+    epipolar_lines = convert_to_homogenous_coordinates(coordinates_i1) @ i2Fi1.T
+    norms = np.linalg.norm(epipolar_lines[:, :2], axis=1, keepdims=True)
+    norms = np.where(norms > 0, norms, 1)
```

Comment:
maybe an inline comment would help here?

Reply:
removed this